### ISSUE TITLE
Syntax is different from VSCode and Python-Markdown

### ISSUE THREAD

**Issue by patarapolw** [docarys/markdown-it-admonition#7]
It's
```
!!! note
    put note here
```
vs
```
!!! note
put note here
!!!
```

I think Mkdocs use the former.

**Comment by berot3** [comment]
thats true, but I have to say I think I like the markdown-it-admonition-style more... but yeah, pymd is maybe a but more established?

**Comment by miffels** [comment]
Same issue here - would be interested in having this as a linter plugin for mkdocs code, but it currently doesn't seem to be useful due to the syntax difference. Maybe an option to support the other format would be useful?

Looking at the history of this repo, though, I wonder if it is still maintained at all.

**Comment by sesispla** [comment]
Hi all,

Long story short, I had trouble in the past with the parser to find the end of the admonition without the trail '!!!'.

Happy to accept a PR and release a new version!

**Comment by sesispla** [comment]
PS: We are dropping docarys support as it stands today, BTW. 

We moved our core documentation system to gatsbyjs, as we can do far more things. The project and the template is proprietary at the moment, but we don't discard release it under the "docarys" brand in the future.
